### PR TITLE
updated code

### PR DESCRIPTION
**Pull request by sayerwalam52**

### CODE REVIEW

File: app/views/material_admin/home.html.erb [hunk 1]
```diff
@@ -60,7 +60,11 @@
             <% end %>
           </li>
           <li>
+<<<<<<< HEAD
+            <%= link_to '/message', method: "get" do %>
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: is this the line you want?

File: app/views/material_admin/home.html.erb [hunk 2]
```diff
@@ -80,7 +84,11 @@
                </li>
               <li>
                 <!-- change to link_to when implementing this part -->
+<<<<<<< HEAD
+                <%= link_to '/message', method: "get" do %>
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: same with this line

File: app/views/material_admin/home.html.erb (hunk 1 above)
Comment (sayerwalam52) [reply app/views/material_admin/home.html.erb]: no i have actually removed those lines

Comment (anagabriel42) [reply app/views/material_admin/home.html.erb]: alright! so the pieces that have 
`<<<<<< HEAD` 
you want to remove until it says
`=======`
then removed the line that says
`>>>>>>> some_git_id`

Comment (anagabriel42) [reply app/views/material_admin/home.html.erb]: don't for get to remove the `======` or your code is going to have syntax errors

File: app/views/material_admin/home.html.erb [hunk 3]
```diff
@@ -60,7 +60,11 @@
             <% end %>
           </li>
           <li>
+<<<<<<< HEAD
+            <%= link_to '/message', method: "get" do %>
+=======
             <%= link_to messages_path do %>
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: leave this line

File: app/views/material_admin/home.html.erb [hunk 4]
```diff
@@ -60,7 +60,11 @@
...
+>>>>>>> ba166d54185e7f55c8ed4b09346d163381c6425c
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: remove this line

File: app/views/material_admin/home.html.erb [hunk 5]
```diff
@@ -80,7 +84,11 @@
                </li>
               <li>
                 <!-- change to link_to when implementing this part -->
+<<<<<<< HEAD
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: remove

File: app/views/material_admin/home.html.erb (hunk 2 above)
Comment (anagabriel42) [reply app/views/material_admin/home.html.erb]: remove

File: app/views/material_admin/home.html.erb [hunk 6]
```diff
@@ -80,7 +84,11 @@
...
+                <%= link_to '/message', method: "get" do %>
+=======
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: remove

File: app/views/material_admin/home.html.erb [hunk 7]
```diff
@@ -80,7 +84,11 @@
...
                 <%= link_to messages_path,  do %>
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: leave this line

File: app/views/material_admin/home.html.erb [hunk 8]
```diff
@@ -80,7 +84,11 @@
...
+>>>>>>> ba166d54185e7f55c8ed4b09346d163381c6425c
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: remove this line

File: app/views/material_admin/home.html.erb [hunk 9]
```diff
@@ -84,11 +81,8 @@
                </li>
               <li>
                 <!-- change to link_to when implementing this part -->
-<<<<<<< HEAD
                 <%= link_to '/message', method: "get" do %>
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: remove this line

File: app/views/material_admin/home.html.erb [hunk 10]
```diff
@@ -60,11 +60,8 @@
             <% end %>
           </li>
           <li>
-<<<<<<< HEAD
             <%= link_to '/message', method: "get" do %>
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: remove this line

File: app/views/material_admin/home.html.erb [hunk 11]
```diff
@@ -80,7 +80,7 @@
                </li>
               <li>
                 <!-- change to link_to when implementing this part -->
-                <%= link_to messages_path,  do %>
+                <%= link_to '/message', method: "get" do %>
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: is this the line that you want?

Comment (sayerwalam52) [reply app/views/material_admin/home.html.erb]: yes

**Pull request merged**